src/main/java/com/softserveinc/ita/pageobjects/SearchPage.java

```diff
@@ -0,0 +1,25 @@
+package com.softserveinc.ita.pageobjects;
+
+import java.util.List;
+
+import static com.softserveinc.ita.utils.runners.ElementsUtil.getListWithGoods;
+
+public class SearchPage extends BasePage {
+
+    public List<String> getGoodsList(String item) {
+        return getListWithGoods("//*[@class='goods-tile__title']", item);
+    }
+
+    public String getFirstRequiredItem(String item) {
```

Comment:
probably operations with getting first and last string from list, could be performed in test